pkg/config/config.go

```diff
@@ -18,11 +18,13 @@ import (
 
 // Config contains a list of rules
 type Config struct {
-	Rules              []*rule.Rule `yaml:"rules"`
-	IgnoreFiles        []string     `yaml:"ignore_files"`
-	SuccessExitMessage *string      `yaml:"success_exit_message"`
-	IncludeNote        bool         `yaml:"include_note"`
-	ExcludeCategories  []string     `yaml:"exclude_categories"`
+	Rules              []*rule.Rule      `yaml:"rules"`
+	IgnoreFiles        []string          `yaml:"ignore_files"`
+	SuccessExitMessage *string           `yaml:"success_exit_message"`
+	IncludeNote        bool              `yaml:"include_note"`
+	ExcludeCategories  []string          `yaml:"exclude_categories"`
+	Labels             map[string]string `yaml:"labels,omitempty"`
+	PushgatewayURL     string            `yaml:"pushgateway"`
```

Comment:
As discussed over email, please don't put these output-specific configs at the top level of woke config. I would prefer something else like 

```yaml
outputs:
  prometheus:
    pushgateway: https://my.url/
    labels:
      repo: sample
      something: else
```